frontend/src/store/settings.ts

```diff
@@ -27,6 +34,13 @@ export default function useSettingsStore() {
       : undefined;
   });
 
+  if(language.value.value===''){
```

Comment:
Reminder to run `yarn prettier`, this formatting seems inconsistent with what prettier outputs

Reply:
My bad, addressed it in https://github.com/ScopeLift/umbra-protocol/pull/331/commits/59f27af3f89430f2f950d32f36fa83d57383e44a